### PR TITLE
Clubs page

### PR DESCRIPTION
**Pull request by Lykhohub**

### CODE REVIEW

**Review by Lykhohub — COMMENTED** [review]
.

**Comment by Markusplay** [comment]
squash all commits into one, and add correct commit messages.

**Comment by Markusplay** [comment]
remove .idea  files from changes

File: app/(client)/clubs/page.tsx [hunk 1]
```diff
@@ -1,5 +1,129 @@
...
+      }
+
+      setFilteredClubs(filtered);
+    };
+
+    filterByCategoryAndSearch();
+  }, [selectedCategory, searchTerm, clubs]);
+
+  const handleShowMore = () => {
+    setVisibleCount(prevCount => prevCount + 6); // Increase visible clubs by 6
```
Comment (Markusplay) [inline app/(client)/clubs/page.tsx:70]: we have 8 clubs per page, fix it and check in other parts

**Comment by Markusplay** [comment]
Add Link on Clubs in header. Check design

File: app/(client)/clubs/page.tsx [hunk 2]
```diff
@@ -1,5 +1,129 @@
...
+            >
+              {category}
+            </Button>
+          ))}
+          <Button
+            variant={selectedCategory === null ? 'outline' : 'default'}
+            className="w-[150px] h-[20px] px-4"
+            onClick={() => setSelectedCategory(null)}
+          >
+            Всі
```
Comment (Markusplay) [inline app/(client)/clubs/page.tsx:94]: there is no "Всі" button in design

File: app/(client)/clubs/page.tsx
Comment (Markusplay) [inline app/(client)/clubs/page.tsx:1]: image
Page is not adaptive. And You must use another component for filters on mobile

File: components/clubs/club-card.tsx
Comment (Markusplay) [inline components/clubs/club-card.tsx:1]: Check your styles for mobile page, and check the design

**Review by Markusplay — CHANGES_REQUESTED** [review]

**Review by Markusplay — APPROVED** [review]

**Pull request merged**